benchmarks/linear_algebra/blas/level2/gemv/gemv_generator.cpp

```diff
@@ -0,0 +1,129 @@
+#include <tiramisu/tiramisu.h>
+
+using namespace tiramisu;
+
+
+/**
+ * Benchmark for BLAS GEMV
+ *     out = a*A*x + b*y
+ *  
+ *
+ *     A : is a M x N matrix
+ *     x : is a size N vector
+ *     y : is a size M vector
+ *     a,b : are scalars
+ *
+ *     out : is a size M vector
+ */
+/**
+We will make a tiramisu implementation of this code :
+  for(int i=0;i<M;i++){
+    tmp=0;
+    for(int j=0;j<N;j++){
+      tmp += A(i,j)*x(j)
+    }
+	tmp *= alpha
+    result(i) = tmp + beta * y(i)
+  }
+
+
+*/
+void generate_function(std::string name)
+{
+    tiramisu::global::set_default_tiramisu_options();
+
+    // -------------------------------------------------------
+    // Layer I
+    // -------------------------------------------------------
+
+    tiramisu::function func(name);
+
+    
+    //Declare both iteration variables
+    tiramisu::var i("i");
+    tiramisu::var j("j");
+	//Matrix Dimensions
+    tiramisu::computation SIZES("{SIZES[k]: 0<=k<2}", tiramisu::expr(), false, p_float64, &func);
```

Comment:
Can you use the new syntax of Tiramisu instead of the old one?

Reply:
You find the new syntax in the first two or three tutorials.

Reply:
Thank you. It's done.